db/migrate/20170704102536_use_deleted_in_containers_tables.rb

```diff
@@ -0,0 +1,55 @@
+class UseDeletedInContainersTables < ActiveRecord::Migration[5.0]
+  class ContainerDefinition < ActiveRecord::Base
+  end
+
+  class ContainerGroup < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class ContainerImage < ActiveRecord::Base
+  end
+
+  class ContainerProject < ActiveRecord::Base
+  end
+
+  class ContainerNode < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Container < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def disconnect_to_soft_delete(model)
+    model.where.not(:deleted_on => nil).update_all("ems_id = old_ems_id")
```

Comment:
I wonder if I should also clean up the case where :ems_id is nil but :deleted_on is also nil? Putting Time.now into deleted_on?